release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -1233,6 +1233,10 @@ With this release, replacement control plane nodes are assigned to the correct i
 
 * Previously, when downloading a `kubeconfig` file for a ServiceAccount, an error was displayed and the ServiceAccount token was unable to be reached. This error was due to the removal of automatically generated secrets. With this update, the download `kubeconfig` action has been removed and the error no longer occurs. (link:https://issues.redhat.com/browse/OCPBUGS-7308[*BZ#OCPBUGS-7308*])
 
+* In the {product-title} 4.12 release, the node terminal tab would display an error because of missing annotations that were caused by pod security measures. Without the required annotations, the node debug pod cannot start. With this update, {product-title} adds these annotations, so that the node debug pod can start and the node terminal tab loads without any errors. (link:https://issues.redhat.com/browse/OCPBUGS-4252)
```

Comment:
A couple suggestions and a few console terminology items
```suggestion
* Previously, the *Terminal* tab on the *Node details* page displayed an error because of missing annotations that were caused by pod security measures. Without the required annotations, the node debug pod cannot start. With this update, {product-title} adds these annotations, so the node debug pod can start and the *Terminal* tab loads without any errors. (link:https://issues.redhat.com/browse/OCPBUGS-4252)
```